spec/fixtures/json/patients/CMS134v6/Pass_Numer.json

```diff
@@ -1,623 +0,0 @@
-{
```

Comment:
This patient was removed because of the ucum unit issue--need to change the "cc" to "cm3" and re-export